doc/changelogs/CHANGELOG_V20.md

```diff
@@ -47,6 +51,24 @@
   * [io.js](CHANGELOG_IOJS.md)
   * [Archive](CHANGELOG_ARCHIVE.md)
 
+<a id="20.9.0"></a>
+
+## 2023-10-24, Version 20.9.0 'Iron' (LTS), @richardlau
+
+### Notable Changes
+
+This release marks the transition of Node.js 20.x into Long Term Support (LTS)
+with the codename 'Iron'. The 20.x release line now moves into "Active LTS"
+and will remain so until October 2024. After that time, it will move into
+"Maintenance" until end of life in April 2026.
+
+### Known issue
+
+* The env var `NODE_V8_COVERAGE` intermittently causes the test runner to hang [#49344](https://github.com/nodejs/node/issues/49344)
+  Collecting code coverage via the `NODE_V8_COVERAGE` environment variable made
```

Comment:
```suggestion
  Collecting code coverage via the `NODE_V8_COVERAGE` environment variable may
```

Reply:
Should we remove the reference to the test runner? As far as I know, the bug is not specific to it.

Reply:
The reference to the testrunner is the current title of https://github.com/nodejs/node/issues/49344. Happy to match if that changes accordingly.

Reply:
Thanks @richardlau!